app/controllers/katello/api/v2/api_controller.rb

```diff
@@ -216,5 +217,44 @@ def check_disconnected
             "is prohibited. If you would like to change this, please update the content setting 'Disconnected mode'."
       fail HttpErrors::BadRequest, _(msg) if Setting[:content_disconnected]
     end
+
+    def filtered_associations
+      nil
+    end
+
+    def filter_association_ids
+      if filtered_associations
```

Comment:
the thought is that we would eventually remove this 'if' and just fail if its not defined.

Reply:
if these are moved to a module then filtered_associations could raise a NotImplementedError